### ISSUE TITLE
Code: access spreadsheets from Gdrive

### ISSUE THREAD

**Issue by Index01** [Index01/RoomBot#39]
It would be nice if we could directly access the rooms and guests spreadsheets(main take3 rooms and secret party export) from drive for production and test. This allows us to bypass the upload a file from and admin and do all the work to sanitize that input. Since we are already using Google credentials for sending email accessing files should be "easy". Then we can upload test/live rooms/guests spreadsheets and use all the nice drive permissions stuff etc. 

Something like this prolly: https://developers.google.com/drive/api/quickstart/python

Auth should be setup before taking in this ticket but if not there is an auth issue with the same guide above. 

Completion criteria:
- ideally functions would be written directly in the backend/reservations/views/admin.py. if not feel comfortable tho can write a standalone module or a script or wtvr.
- call function, get spreadsheet in a format that works with existing create_guests function.
- get prod and dev spreadsheets
- probably need to create directories in drive. Do something clever like /test & /prod or similar.

**Comment by Index01** [comment]
Prerequisite: issue #38

**Comment by mpesavento** [comment]
This is a low priority task. 

Having the original room allocations as a CSV will be sufficient.
I'd like to know how we will be merging in the existing staff/manually allocated guests into the target rooms.

**Comment by Index01** [comment]
hiiiiighhhhhest priority in my opinion. 
we must has a way to upload guest spreadsheet(aka secret party export)

can describe btr when not 2am

**Comment by otakup0pe** [comment]
Given google sheets are no longer a source of truth, we no longer need this functionality.
